packages/front-end/components/Experiment/TabbedPage/HealthTab.tsx

```diff
@@ -49,6 +58,17 @@ export default function HealthTab({
   const [setupModalOpen, setSetupModalOpen] = useState<boolean>(false);
   const [loading, setLoading] = useState<boolean>(false);
 
+  const healthTabConfigParams: HealthTabConfigParams = {
+    experiment: experiment,
+    phase: phase,
+    exposureQueryDimensions: exposureQuery?.dimensions || [],
+    refreshOrganization: refreshOrganization,
+    mutateSnapshot: mutateSnapshot,
+    setAnalysisSettings: setAnalysisSettings,
+    setLoading: setLoading,
+    resetResultsSettings: resetResultsSettings,
+  };
```

Comment:
Nit: for object properties whose values have the same name as the value expected by the type you can omit the assignment portion (i.e. setLoading: setloading -> setLoading)
```suggestion
  const healthTabConfigParams: HealthTabConfigParams = {
    experiment,
    phase,
    exposureQueryDimensions: exposureQuery?.dimensions || [],
    refreshOrganization,
    mutateSnapshot,
    setAnalysisSettings,
    setLoading,
    resetResultsSettings,
  };
```